### DISCUSSION TITLE
Why not join the HA-LCARS project?

### DISCUSSION THREAD

**Discussion by veedubb65** [snootched/cb-lcars#2] [Ideas]
Have you considered trying to join the HA-LCARS project?

You've added some really great stuff, but not being available through HACS and requiring manual changes to files that are part of HA-LCARS means that the whole thing will break whenever HA-LCARS is updated. I would really love to see a single addon theme that includes all of the LCARS functionality in one place.

**Comment by snootched** [comment]
Thanks very much for your feedback, appreciate it.  This is my first crack at something like this, so glad you like it.

I understand your concern here.  While I'm not opposed to it - from what I understand about HA-LCARS is that the preference when creating it was to keep it pretty lightweight and minimize external dependencies.  Since it's a card-mod based theme, that is doable within the theme file.

This one, is not so much a theme as it is a collection of templates (configurations) to control other custom components.  It 'installs' in a different manner since this is the way custom button-card works.  The templates cannot be put into the theme file (from what I can tell anyway..) so I'm not sure how to easily add it to HA-LCARS.

I could potentially decouple it from HA-LCARS in a way so that it's not broken if something happens to HA-LCARS, I have to think about it a bit.  Really relying on HA-LCARS to set up all the theme variables (primarily the colour variables.)  I like to use more colurs (and differen names) than what is present in the current version, so these just pust into the custom section where people like to put their own mods.  Currently, this is overwritten any time HA-LCARS is updated from HACS.

Happy to hear any ideas, and I'll give it a think on how to make it a bit more resilient.
